notebooks/rbc-omnisci-ibis.ipynb

```diff
@@ -2,39 +2,39 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
     "import ibis\n",
-    "from rbc.mapd import RemoteMapD"
+    "from rbc.omniscidb import RemoteOmnisci"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [],
    "source": [
-    "mapd = RemoteMapD()  # RemoteMapD(user=, password=, host=, port=, dbname=)\n",
-    "mapd_ibis = ibis.mapd.connect(user=mapd.user, password=mapd.password,\n",
-    "                              host=mapd.host, port=mapd.port, database=mapd.dbname)"
+    "omni = RemoteOmnisci()  # RemoteOmisci(user=, password=, host=, port=, dbname=)\n",
+    "omni_ibis = ibis.omniscidb.connect(user=omni.user, password=omni.password,\n",
```

Comment:
`ibis` does not have `omniscidb`. Restore it as `mapd`.

Reply:
Hi @pearu, `mapD` was renamed to `omniscidb` on ibis 6 months ago. See:
https://github.com/ibis-project/ibis/tree/master/ibis/omniscidb
https://github.com/ibis-project/ibis/commit/9df27958f305a901728b540200bd8fa2820d4625

Reply:
Ok, let's keep ibis.omniscidb. (Todo for myself: update rbc-dev environment).

Reply:
I had to install ibis from master to get the latest changes: `pip install git+https://github.com/ibis-project/ibis@master`